### ISSUE TITLE
doesn't use other camera when main camera is disable 

### ISSUE THREAD

**Issue by gun4qmm7h** [moezbhatti/qksms#1823]
### DESCRIPTION
A short description of the issue. 
Doesn't use open camera and if main camera is disabled then it doesn't work when trying to take a picture.

### STEPS
Disable the camera
go to qksms and try to take a photo

### EXPECTED
uses any camera app or ask what camera app to use

### OBSERVATIONS
shows a message saying that "No apps can perform this action"

**Comment by AgentGibbz** [comment]
I'm here to report the same issue. I use an alternative camera app, and when I try to take a photo within QKSMS it gives the message "No apps can perform this action"

So the only way for me to send a photo is to go open my camera app of choice, take a photo, then come back to QKSMS and attach the photo. It's not as convenient. Would love for this to be fixed.

**Comment by JonTabor** [comment]
Yep, same issue for me.  Camera that I'm using is MWP's Google Camera 8.4.300, which is based off the stock Google Camera app.
